assets/src/scss/new-identity/style.scss

```diff
@@ -9,4 +9,22 @@
   --block-columns--step-number--background: var(--gp-green-100);
   --block-columns--step-number--color: var(--grey-900);
   --block-columns--card-header--background: var(--p4-dark-green-800);
+  --link--focus--outline: 2px solid #99c3ff;
+  --top-page-tags--tag-item--background: var(--gp-green-100);
```

Comment:
This one feels unrelated?

Reply:
This was part of the previous implementation. I forgot to remove it 🙏

Reply:
You marked this as resolved, but the `--top-page-tags--tag-item--background` override is still here 🤔

Reply:
Sorry, I forgot to push it 🤦